apptail/storage/tail_tracker.go

```diff
@@ -0,0 +1,173 @@
+package storage
+
+import (
+	"fmt"
+	"github.com/ActiveState/log"
+	"runtime"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+type Tracker interface {
+	Update(instKey string, childKey string, childVal int64)
+	LoadTailers()
+	Remove(key string)
+	RegisterInstance(instKey string)
+	InitializeChildNode(instKey string, childkey string, offSet int64)
+	Submit()
```

Comment:
I think I'd prefer this to be called `Commit()` rather than `Submit()`, since that's what it actually does - it commits the changes to disk.